scapy/packet.py

```diff
@@ -850,6 +850,34 @@ def loop(todo, done, self=self):
             done = {}
         return loop(todo, done)
 
+    def __iterlen__(self):
+        """Predict the total length of the iterator"""
+        fields = [k for (k, v) in itertools.chain(six.iteritems(self.default_fields),
+                  six.iteritems(self.overloaded_fields))
+                  if isinstance(v, VolatileValue)] + list(self.fields.keys())
+        l = 1
```

Comment:
Could you use variable length other than l, f, or v? I hope that I will manage to remove all flake8 warnings some day =)